srcpkgs/qt5/template

```diff
@@ -425,7 +428,6 @@ do_install() {
 	cd ${wrksrc}/qtbase
 	make INSTALL_ROOT=${DESTDIR} install
 
-	vlicense ${wrksrc}/LGPL_EXCEPTION.txt
```

Comment:
Shouldn't we include the LGPL_EXPECTION.txt or was it removed?

Reply:
File was removed

Reply:
Readded it from one of the sub dir

Reply:
Well, I tried to build your PR and also realized it didn't work. I guess we need to find out why the file was removed.

Reply:
Ok, here's the commit removing it:
https://code.qt.io/cgit/qt/qt5.git/commit/?h=dev&id=21a37fe009d28e394d154c4069de332075814f8f

This means we need to update `license=` to use `LGPL-3.0-or-later`

Reply:
> Well, I tried to build your PR and also realized it didn't work.

It should work fine for at least x86_64 glibc. Haven't gotten cross builds figured out yet. I work currently on getting webengine patched for musl